roles/reboot/tasks/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
+# provide an empty iterator when a list is not provided
+# http://docs.ansible.com/ansible/playbooks_conditionals.html#loops-and-conditionals
 - include: ../../../common/ans_reboot.yaml
+  vars:
+    services: "{{ wait_for_services|default([]) }}"
```

Comment:
Hmm, and if you completely omit the `vars` section? Does it pass through variables from higher up to reboot?